KaijuKiller/Assets/Scripts/PlayerMove.cs

```diff
@@ -48,6 +54,16 @@ void Update()
                 //bounds += 1; //bug
             }
         }
+
+        if(isJumping == false)
+        {
+            if(Input.GetKeyDown(KeyCode.Space))
+            {
+                //rb.velocity = new Vector3(0, jumpSpeed, moveSpeed);
+                //isJumping = true;
+                //StartCoroutine(stopJump());
```

Comment:
This function is commented, why?

Reply:
Also consider adding a layer of SoC to your update function, pull blocks of code out into their own function to increase maintainability

Reply:
currently theres a bug that stops the player mid movement when jumping AND moving